README.md

```diff
@@ -83,8 +86,8 @@ The [project factory](https://github.com/terraform-google-modules/terraform-goog
 
 ### Software Dependencies
 ### Terraform
-- [Terraform](https://www.terraform.io/downloads.html) 0.11.x
-- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v2.0.0
+- [Terraform](https://www.terraform.io/downloads.html) >= 0.12.0
```

Comment:
Not sure, but I think that `~ 0.12.0` is better

Reply:
We're looking at version greater or equal to 0.12 >= looks  more accurate to me